scivision/catalog/catalog.py

```diff
@@ -28,7 +28,7 @@ class CatalogModelEntry(BaseModel, extra="forbid"):
     format: str
     pretrained: bool
     labels_required: bool
-    institution: Optional[str]
+    institution: Optional[Tuple[str, ...]]
```

Comment:
`Optional[Tuple[str, ...]]` means that both `None` and `()` are valid inputs. Do we really need both? How about just `Tuple[str, ...]`, with a default of `()` so it isn't required in the catalog entry?

Reply:
Ok, have done, btw just out of curiosity, why do we use "Tuple" here when it seems like what we actually have is a list?

Reply:
A list would certainly be clearer - a tuple is being used here as a substitute for an *immutable* list. (Even better might be a frozenset, since we don't care about the order of the items, or duplicates).  Being immutable means it's hashable and can be more convenient when included in a dataframe. This probably deserves a comment - I can add one.